### PR TITLE
Add randomcolor

### PR DESCRIPTION
**Pull request by hadim**

### CODE REVIEW

**Comment by hadim** [comment]
LGTM @jakirkham and @ocefpaf

File: recipes/randomcolor/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+test:
+  imports:
+    - randomcolor
+  commands:
+    - cd $SRC_DIR && python setup.py test  # [unix]
+
+about:
+  home: https://github.com/kevinwuhoo/randomcolor-py
+  license: GPLv3
```
Comment (jakirkham) [inline recipes/randomcolor/meta.yaml]: Please ask upstream to include a license file and add it to `MANIFEST.in`.

File: recipes/randomcolor/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - setuptools
```
Comment (jakirkham) [inline recipes/randomcolor/meta.yaml]: `setuptools` shouldn't be needed at run time for this package.

File: recipes/randomcolor/meta.yaml (hunk 1 above)
Comment (hadim) [inline recipes/randomcolor/meta.yaml]: Will do.

File: recipes/randomcolor/meta.yaml (hunk 2 above)
Comment (hadim) [inline recipes/randomcolor/meta.yaml]: Correct !

File: recipes/randomcolor/meta.yaml (hunk 1 above)
Comment (hadim) [inline recipes/randomcolor/meta.yaml]: Well in fact setuptools is needed to install the package for testing....

File: recipes/randomcolor/meta.yaml (hunk 2 above)
Comment (hadim) [inline recipes/randomcolor/meta.yaml]: Well in fact setuptools is needed to install the package for testing....

Comment (hadim) [inline recipes/randomcolor/meta.yaml]: Look here https://circleci.com/gh/conda-forge/staged-recipes/10858?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Comment (hadim) [inline recipes/randomcolor/meta.yaml]: Should I include setuptools or do you prefer I move the test during the build ?

**Comment by hadim** [comment]
LGTM here @jakirkham

**Comment by hadim** [comment]
Waiting for https://github.com/pelson/Obvious-CI/pull/67 before to merge.

**Comment by hadim** [comment]
LGTM @ocefpaf : could you merge this please ?

**Comment by ocefpaf** [comment]
It is all good here. Let me just merge https://github.com/conda-forge/conda-smithy-feedstock/pull/50 first so the feedstock is created properly.

**Pull request merged**